service/templates/admin.html

```diff
@@ -822,6 +823,25 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 									</md-input-container>
 								</md-card-content>
 							</md-card>
+            </section>
+
+            <section layout="row" flex>
```

Comment:
Can this go next to the jira section in the right column? There is an unnecessary hole there right now

Reply:
done